index.css

```diff
@@ -287,39 +333,77 @@ h4 {
         justify-content: space-around;
         order: 1;
     }
-}
 
-/* iframe {
-    width: 100%;
-    height: 77px;
-    border:0;
-    transform: scale(1);
-    overflow:hidden;
-} */
-
-.doc-container {
-    position: relative;
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-start;
-    margin: 2rem 12rem;
-}
+    .doc-container {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin: 2rem 8rem;
+    }
 
-.side-nav {
-    height: 100%;
-    flex-basis: 30vw;
-    position: sticky;
-    top: 0em;
-}
+    .side-nav {
+        height: 100%;
+    }
+        
+    .doc-content {
+        margin: 0 4rem;
+        flex-basis: 100vw;
+    }
 
-.side-nav-items {
-    font-size: 1.5rem;
-    margin: 1rem;
+    #get-started-expand-more {
+        display: none;
+    }
+
+    #components-expand-more {
+        display: none;
+    }
+
+    #get-started-expand-less {
+        display: none;
+    }
+
+    #components-expand-less {
+        display: none;
+    }
```

Comment:
maybe u could use classes here for reusability and again comma separation so that u don't have to write repeatedly the same property